.github/workflows/build-image.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: Build and Publish ClamAV Image
+on:
+    workflow_dispatch:
+    workflow_call:
+
+env:
+  DOCKER_NAME: clamav-rest
+  IMAGE: clamav
+  GH_REPO: gsa-tts/fac
+
+jobs:
+  build-clamav:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Get Date
+        shell: bash
+        id: date
+        run: |
+          echo "date=$(date +%Y%m%d)" >> $GITHUB_OUTPUT
+
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Build Docker Image
+        run: docker build . -t ${{ env.DOCKER_NAME }}:${{ steps.date.outputs.date }}
+
+      - name: Tag Image
+        run: |
+          docker tag ${{ env.DOCKER_NAME }}:${{ steps.date.outputs.date }} ghcr.io/${{ env.GH_REPO }}/${{ env.IMAGE }}:${{ steps.date.outputs.date }}
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push Image
+        run: docker push --all-tags ghcr.io/${{ env.GH_REPO }}/${{ env.IMAGE }}
+
+      - name: Upload Artifact
```

Comment:
What's this for? Caching for later runs?

Reply:
And reusing the artifact, can remove it if we dont feel like we need it, just noticed our base scan-images.yml creates the artifact.